internal/store/node_test.go

```diff
@@ -46,6 +46,7 @@ func TestNodeStore(t *testing.T) {
 				},
 				Spec: v1.NodeSpec{
 					ProviderID: "provider://i-uniqueid",
+					PodCIDR:    "172.24.10.0/24",
```

Comment:
Since this can be empty or not set, lets have one test where this is not set.

Reply:
I'm adding a general test case with an empty `v1.Node` and verifying all empty strings. Not sure if you have preferences over the ordering of test cases but I'm happy to change that.